src/components/theharvester/theharvester.tsx

```diff
@@ -99,9 +133,27 @@ const TheHarvester = () => {
             CommandHelper.runCommand("kill", args);
         }
     };
+
+    // Handles saving the output after the tool use
+    const handleSaveComplete = () => {
+        // Indicates  the file has been saved and passed into the SaveOutputToTextFile function
+        setHasSaved(true);
+        setAllowSave(false);
+    };
+
+    /**
+     * onSubmit: Asynchronous handler for the form submission event.
+     * It sets up and triggers the airbase-ng tool with the given parameters.
+     * Once the command is executed, the results or errors are displayed in the output.
+     *
+     * @param {FormValuesType} values - The form values, containing the fake host name, channel, and WLAN interface.
```

Comment:
Need to change these form values to match the actual values in this tool. 
```suggestion
     * @param {FormValuesType} values - The form values, containing the domain, result limit, etc.
```